xml/admin_saltcluster.xml

```diff
@@ -1215,6 +1215,102 @@ OSD_ID  HOST         STATE                    PG_COUNT  REPLACE  FORCE  STARTED_
+    new one - for example, when OSDs have different DB sizes and you need to align them.
+   </para>
+   <tip>
+    <title><command>ceph-volume</command> naming convention</title>
+    <para>
+     Some clusters may have old volume group (VG) or logical volume (LV) names prefixed with
+     <literal>ceph-block-dbs</literal> and <literal>osd-block-db</literal>, for example:
+    </para>
+<screen>ceph-block-dbs-c3dc9227-ca3e-49bc-992c-00602cb3eec7/osd-block-db-b346b9ff-dbbe-40db-a95e-2419ccd31f2c</screen>
+    <para>
+     The current naming convention is as follows:
+    </para>
+<screen>ceph-c3dc9227-ca3e-49bc-992c-00602cb3eec7/osd-db-b346b9ff-dbbe-40db-a95e-2419ccd31f2c</screen>
+   </tip>
+   <procedure>
+    <title>Migrating a DB device to a new device</title>
+    <step>
+     <para>
+      Identify the <option>db device</option> and <option>osd fsid</option> values by running
+      the following command:
+     </para>
+<screen>&prompt.cephuser;cephadm ceph-volume lvm list
+[...]
+====== osd.0 =======
+
+[block]       /dev/ceph-b03b5ad4-98e8-446a-9a9f-840ecd90215c/osd-block-c276d2a4-5578-4847-94c6-8e2e6abf81c4
+
+block device              /dev/ceph-b03b5ad4-98e8-446a-9a9f-840ecd90215c/osd-block-c276d2a4-5578-4847-94c6-8e2e6abf81c4
+block uuid                Kg3ySP-ykP8-adFE-UrHY-OSiv-0WQ5-uuUEJ9
+cephx lockbox secret
+cluster fsid              9c8d3126-9faf-11ec-a2cf-52540035cdc1
+cluster name              ceph
+crush device class
+db device                 /dev/ceph-block-dbs-c3dc9227-ca3e-49bc-992c-00602cb3eec7/osd-block-db-b346b9ff-dbbe-40db-a95e-2419ccd31f2c
+encrypted                 0
+osd fsid                  c276d2a4-5578-4847-94c6-8e2e6abf81c4
+osd id                    0
+osdspec affinity          sesdev_osd_deployment
+type                      block
+vdo                       0
+devices                   /dev/vdb
+[...]</screen>
+    </step>
+    <step>
+     <para>
+      Create a new logical volume (LV) for the new DB device. Refer to
+      <xref linkend="rec-waldb-size"/> when determining the right size for the DB device. For
+      example:
+     </para>
+<screen>&prompt.root;lvcreate -n osd-db-$(cat /proc/sys/kernel/random/uuid) \
+ ceph-c3dc9227-ca3e-49bc-992c-00602cb3eec7 --size <replaceable>DB_SIZE</replaceable></screen>
+    </step>
+    <step>
+     <para>
+      Stop the OSD. Run the following command on the OSD node where the OSD daemon runs:
+     </para>
+<screen>&prompt.cephuser.osd;cephadm unit --name osd.0</screen>
+    </step>
+    <step>
+     <para>
+      Enter the shell on the stopped OSD container:
+     </para>
+<screen>&prompt.cephuser.osd;cephadm shell --name osd.0</screen>
+    </step>
+    <step>
+     <para>
+      Migrate the DB device:
+     </para>
+<screen>&prompt.cephuser;ceph-volume lvm migrate --osd-id 0 \
+ --osd-fsid c276d2a4-5578-4847-94c6-8e2e6abf81c4 --from db \
+ --target ceph-c3dc9227-ca3e-49bc-992c-00602cb3eec7/osd-db-b346b9ff-dbbe-40db-a95e-2419ccd31f2c</screen>
```

Comment:
@lezare  Don't we need `ceph-volume lvm new-db` before `lvm migrate`? I did not test these instructions but AFAIR when I did migration I needed to run `new-db`.

[1] https://docs.ceph.com/en/quincy/ceph-volume/lvm/newdb/

Reply:
Or may be it was a little different case of migrating from osd withoud dedicated db...

Anyway I am fine with changes if they are tested and proved to work. Otherwise I will need to test it before approval.